php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -309,10 +320,15 @@ class NDB_BVL_Instrument extends NDB_Page
      * @access   public
      * @abstract
      */
+    /*
```

Comment:
This can be removed

Reply:
I looked at some instruments in IBIS, and it turns out that it can't be. PHP instruments are written assuming it exists (and including logic in it), so the call needs to remain in the factory, and the factory needs it to be defined in the base class so that it doesn't fail it an instrument doesn't implement it.

We should get rid of it long term, but today is not the day...